hapi-server/src/main/java/com/example/fhirexample/utils/LocationRestultParser.java

```diff
@@ -0,0 +1,37 @@
+package com.example.fhirexample.utils;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+
+import org.hl7.fhir.r4.model.Location;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+public class LocationRestultParser {
```

Comment:
I'd love to know what a "Restult" is, never heard of one before. Should probably be "Result" instead.

Reply:
If I had to guess, a Restult is the result of resting, which is not what I was going for. This is fixed now.